packages/gitbook/src/components/RootLayout/CustomizationRootLayout.tsx

```diff
@@ -151,6 +151,7 @@ export async function CustomizationRootLayout(props: {
                     }
                 `}</style>
             </head>
+            <HolyLoader color="rgb(var(--primary-9))" height={1} />
```

Comment:
Honestly I never liked these loaders. They're not very apparent and user friendly 😅. Why not try to use the skeleton grid we had in v1? It looks much better!

Reply:
I wanted to submit a PR with it to see if it works.

Now that we are sure we can detect the loading of a page we can adapt to do whatever we want!